generate_gpg

```diff
@@ -28,5 +28,5 @@ cat > $GPG_SETTINGS <<EOF
 EOF
 gpg2 --homedir $KEYDIR --batch --gen-key $GPG_SETTINGS
 
-echo "Now set GPGKEY and FULLGPGKEY in releases/$VERSION/settings"
+echo "Now set FULLGPGKEY in releases/$VERSION/settings"
 gpg2 --homedir "$KEYDIR" --list-key "$SIGNER"
```

Comment:
Future improvement would be to automatically parse this output and append it to the settings.